src/osd/OSD.cc

```diff
@@ -3775,6 +3775,8 @@ int OSD::init()
       superblock.cluster_osdmap_trim_lower_bound = superblock.get_oldest_map();
     }
 
+    last_trim_maps = ceph_clock_now();
```

Comment:
last_trim_maps should be updated after the call to trim_maps()

Reply:
okay, my previous thought was that after OSD::init, do not call OSD::trim_maps immediately in OSD::tick. @Matan-B